lib/engine/round/base.rb

```diff
@@ -153,7 +153,8 @@ def operating?
 
       def skip_steps
         @steps.each do |step|
-          next if !step.active? || !step.blocks? || @entities[@entity_index]&.closed?
+          step.inactive = !step.active?
```

Comment:
`skip_steps` is called every time an action is processed, so an `@inactive` value won't overstay its welcome

https://github.com/tobymao/18xx/blob/891ed4f699ab5f4bea0d950775c1ec9b29e4412c/lib/engine/round/base.rb#L97